tensorflow_lite_support/cc/task/audio/utils/wav_io.cc

```diff
@@ -113,10 +113,10 @@ absl::Status ReadString(const std::string& data, int expected_length,
 
 absl::Status DecodeLin16WaveAsFloatVector(const std::string& wav_string,
                                           std::vector<float>* float_values,
+                                          int offset,
```

Comment:
Please use `uint32_t` to align with the type of sample_count

Reply:
The original data type for the offset was `int` though.

https://github.com/tensorflow/tflite-support/blob/72b4ca66479054d0463a0e52b59e7a7b2da8ecc0/tensorflow_lite_support/cc/task/audio/utils/wav_io.cc#L119

Reply:
Could you change it to uint32_t? It's better than int because offset can't be negative.

I think it'll require you to change the datatype for offset of underlying methods but I think it makes more sense to align the datatype of offset with that of sample_count through out the codebase.

Reply:
Alrighty, I'll make those changes.